features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature

```diff
@@ -0,0 +1,46 @@
+Feature: git ship: trying the ship of the current feature branch without editing the default commit message
+
+  As a developer shipping a branch
+  I want the ship to abort if I don't edit the default commit message
+  So that I don't have ugly commits merged into my main branch
+
+
+  Background:
+    Given I have a feature branch named "feature"
+    And the following commit exists in my repository
+      | BRANCH  | LOCATION | MESSAGE        | FILE NAME    | FILE CONTENT    |
+      | feature | local    | feature commit | feature_file | feature content |
+    And I am on the "feature" branch
+    When I run `git ship` and don't change the default commit message
+
+
+  Scenario: result
+    Then it runs the Git commands
+      | BRANCH  | COMMAND                            |
+      | feature | git fetch --prune                  |
+      |         | git checkout main                  |
+      | main    | git rebase origin/main             |
+      |         | git checkout feature               |
+      | feature | git merge --no-edit origin/feature |
+      |         | git merge --no-edit main           |
+      |         | git checkout main                  |
+      | main    | git merge --squash feature         |
+      |         | git commit                         |
+      |         | git reset --hard                   |
+      |         | git checkout feature               |
+      | feature | git checkout main                  |
+      | main    | git checkout feature               |
+    And I get the error "Aborting ship due to empty commit message"
+    And I am still on the "feature" branch
+    And I still have the following commits
+      | BRANCH  | LOCATION | MESSAGE        | FILE NAME    | FILE CONTENT    |
+      | feature | local    | feature commit | feature_file | feature content |
+
+
+  Scenario: undo
```

Comment:
This should probably be moved to shared. Checking that you can't undo after an unsuccessful command